actions/mistral_branch_candidate.yaml

```diff
@@ -30,7 +30,7 @@ parameters:
   requirements:
     type: string
     description: List of python dependencies from pip freeze.
-    default: ""
+    default: ''
```

Comment:
what's the difference between using double quotes and single quotes?

Reply:
In this case there is no difference. This is a result of the `ruamel` parser used in `orquestaconvert`.